spec/controllers/application_templates_controller_spec.rb

```diff
@@ -120,7 +121,7 @@
       end
       it 'redirects back' do
         submit
-        expect(response).to redirect_to :back
+        expect(response).to redirect_to 'http://test.host/redirect'
```

Comment:
Nah.  That line was hidden from the diff and I assumed it was some sort of magic-test-value. Looking at the whole test  context it's clear enough.